components/eventType.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+
+export default () => {
+
+    const [eventType, setEventType] = useState([]);
+
+    const { uuid } = useParams();
+
+    const fetchData = async () => {
+        const result = await fetch(
+            `/api/event_types/${uuid}`
+        ).then((res) => res.json());
+
+        setEventType(result.event)
+    }
+
+    console.log(eventType)
```

Comment:
Probably want to remove this.

Reply:
Aw, I thought I removed all console.log's. Sorry about that. I specifically searched for them, but must have missed this one.